arviz/plots/khatplot.py

```diff
@@ -136,7 +137,7 @@ def plot_khat(
     hlines_kwargs.setdefault("linestyle", [":", "-.", "--", "-"])
     hlines_kwargs.setdefault("alpha", 0.7)
     hlines_kwargs.setdefault("zorder", -1)
-    hlines_kwargs.setdefault("color", "C1")
+    hlines_kwargs.setdefault("color", to_hex("C1"))
```

Comment:
After we have managed to generalize vectorized to hex, we could modify this to:

```
hlines_kwargs.setdefault("color", "C1")
hlines_kwargs["color"] = vectorized_to_hex(hlines_kwargs["color"])  # color can be anything accepted by hlines
```

which would bring us closer to using the same colors in bokeh and in matplotlib which is the motivation of converting the colors to hex, matplotlib already works fine.